app/blueprints/fund_builder/routes.py

```diff
@@ -66,12 +73,34 @@
 )
 
 
+@build_fund_bp.route("/healthcheck")
+def healthcheck():
+    return "OK", 200
```

Comment:
Should/could we use the [healthcheck from utils](https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/healthchecks/healthcheck.py) instead

Reply:
I did look at that but was unsure on the benefits so preferred the simpler approach. Can we defer this to a separate ticket if needed?

Reply:
I guess - but that seems like creating more work?

The utils healthcheck is how we do standard healthchecks - why not use it from the get go? It feels good to be consistent from scratch, especially when there's an option available immediately.